incubator/hnc/pkg/kubectl/tree.go

```diff
@@ -93,5 +110,20 @@ func nameAndFootnotes(hier *api.HierarchyConfiguration) string {
 }
 
 func newTreeCmd() *cobra.Command {
+	treeCmd.Flags().BoolP("all-namespaces", "A", true, "Displays all trees on the cluster")
```

Comment:
Doesn't this default the flag to true?

Reply:
Yes default value is true, Should I make it false @rjbez17 ? Actually I am not clear with its context :(

Reply:
Good catch Ryan. Yes, the default should be false. If the default is true, then `kubectl hnc tree` will be the same as `kubectl hnc tree -A` which is _not_ what we want.